site/docs/v1/tech/lambdas/index.adoc

```diff
@@ -46,6 +46,18 @@ The `log`, `info` and `error` will always cause Event Log entries to be created
 
 Message of each type are accumulated during the lambda invocation and a maximum of one event log of each type will be created as a result of the lambda invocation. This means making multiple requests to `console.info` in the lambda function body will result in a single event log of type Information.
 
+When logging objects, you'll need to stringify them to see their data.
+
+[source,javascript]
+----
+function populate(jwt, user, registration) {
+  //...
+  console.log(user); // doesn't log any data other than the fact a user is an object. Probably not what you want.
+  console.log(JSON.stringify(user)); // outputs all the properties of the user object.
```

Comment:
If you want to also provide an example of pretty printing the object:

```
console.log(JSON.stringify(user, null, ' '));
```

Reply:
Awesome, thanks!